compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/FlushCoroutineDispatcher.skiko.kt

```diff
@@ -68,7 +68,7 @@ internal class FlushCoroutineDispatcher(
      * Does the dispatcher have any tasks scheduled or currently in progress
      */
     fun hasTasks() = synchronized(tasksLock) {
-        tasks.isNotEmpty() && delayedTasks.isNotEmpty()
+        tasks.isNotEmpty() || delayedTasks.isNotEmpty()
```

Comment:
Important implementation change:
we had a wrong condition

Reply:
I removed this change from this PR. Here is a new PR with this change - https://github.com/JetBrains/androidx/pull/370